ladybug_vtk/_extend_ladybug_display_schema.py

```diff
@@ -0,0 +1,26 @@
+"""Add capability to turn a Ladybug geometry object into a VTK polydata object."""
+
+from ladybug_display_schema.geometry2d import Point2D, LineSegment2D, Polyline2D, Mesh2D
+from ladybug_display_schema.geometry3d import Point3D, LineSegment3D, Polyline3D, \
+    Arc3D, Mesh3D, Face3D, Polyface3D, Cone, Sphere, Cylinder
+from .from_geometry import from_line2d, from_point2d, from_point3d, from_line3d, \
+    from_polyline3d, from_arc3d, from_mesh3d, from_mesh2d, from_face3d, from_polyface3d, from_cone,\
+    from_sphere, from_cylinder, from_polyline2d
+
+# 2d goemetry
+Point2D.to_polydata = from_point2d
+LineSegment2D.to_polydata = from_line2d
+Polyline2D.to_polydata = from_polyline2d
+
+# 3d geometry
+Point3D.to_polydata = from_point3d
+LineSegment3D.to_polydata = from_line3d
+Polyline3D.to_polydata = from_polyline3d
+Arc3D.to_polydata = from_arc3d
+Mesh3D.to_polydata = from_mesh3d
+Mesh2D.to_polydata = from_mesh2d
+Face3D.to_polydata = from_face3d
+Polyface3D.to_polydata = from_polyface3d
+Cone.to_polydata = from_cone
+Sphere.to_polydata = from_sphere
+Cylinder.to_polydata = from_cylinder
```

Comment:
It's really great to have all of these methods but I'm not sure if code injection is really the most appropriate for this case. Ladybug-geometry does not search for extension packages the way that honeybee-core does. So the behavior of this code injection is going to be different from the way it works in all of our other libraries in that these injected methods won't exist until ladybug-vtk is imported. What if we just had people use these methods directly similar to how we use them in the [`fromgeometry` module of ladybug-rhino](https://github.com/ladybug-tools/ladybug-rhino/blob/master/ladybug_rhino/fromgeometry.py)?